lisa/sut_orchestrator/azure/common.py

```diff
@@ -118,6 +118,7 @@ class AzureNodeSchema:
     # It decides the real computer name. It cannot be too long.
     short_name: str = ""
     vm_size: str = ""
+    osdisk_size_in_gb: int = 30
```

Comment:
Add a comment above to explain it's the min size. If there are other requirements on osdisk size, it will be bigger.

Reply:
Done.

Reply:
to be frank, I am still confused from the comment.

Reply:
Which part are you confused about?

Reply:
@LiliDeng ^^^